locales/bn/json.json

```diff
@@ -51,19 +51,19 @@
     "All resources loaded.": "সবগুলি রিসোর্স লোড করা হয়েছে।",
     "All rights reserved.": "সমস্ত অধিকার সংরক্ষিত।",
     "All subscription plan prices are excluding applicable VAT.": "সমস্ত সাবস্ক্রিপশন প্ল্যান মূল্য প্রযোজ্য ভ্যাট ব্যতীত।",
-    "All subscription plan prices include applicable VAT.": "সমস্ত সদস্যতা পরিকল্পনা মূল্য প্রযোজ্য VAT অন্তর্ভুক্ত.",
+    "All subscription plan prices include applicable VAT.": "সমস্ত সাবস্ক্রিপশন পরিকল্পনা মূল্য প্রযোজ্য VAT অন্তর্ভুক্ত.",
```

Comment:
I think the word "সদস্যতা" was the right one instead of "সাবস্ক্রিপশন". The word "সাবস্ক্রিপশন" is the bengali pronunciation of the English word 'subscription'. Let us know your opinion.